addons/arsenal/functions/fnc_fillRightPanel.sqf

```diff
@@ -86,7 +90,7 @@ private _compatibleMagazines = [[[], []], [[], []], [[], []]];
             // Magazine groups
             {
                 private _magazineGroups = uiNamespace getVariable [QGVAR(magazineGroups),["#CBA_HASH#",[],[],[]]];
-                private _magArray = [_magazineGroups, toLower _x] call CBA_fnc_hashGet;
+                private _magArray = [_magazineGroups, _x] call CBA_fnc_hashGet;
```

Comment:
Again, you cannot rely on the name inside "magazineWell" to be correct case.

Also on a seperate note, it might make sense to convert this stuff to "real" hashmaps